projects/plugins/jetpack/modules/subscriptions/subscribe-overlay/subscribe-overlay.css

```diff
@@ -0,0 +1,83 @@
+body.jetpack-subscribe-overlay-open {
+	overflow: hidden;
+}
+
+.jetpack-subscribe-overlay {
+	--jetpack-subscribe-overlay--text-color: var(--wp--preset--color--primary, black);
+	--jetpack-subscribe-overlay--background-color: var(--wp--preset--color--background, var(--wp--preset--color--base, var(--wp--preset--color--contrast, #F9F9F9)));
+
+	visibility: hidden;
+	position: fixed;
+	z-index: 50000; /* Same as WP.com Action bar */
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	overflow: hidden;
+	background-color: transparent;
+	transition: all 0.4s;
+	color: var(--jetpack-subscribe-overlay--text-color);
+}
+
+.jetpack-subscribe-overlay__content {
+	position: relative;
+	visibility: hidden;
+	overflow: hidden;
+	top: 100%;
+	margin: 15% auto;
+	width: 100%;
+	max-width: 600px;
+	transition: all 0.4s;
+	text-wrap: pretty;
+}
+
+.jetpack-subscribe-overlay__close {
+	display: none;
+	cursor: pointer;
+	position: absolute;
+	top: 32px;
+	right: 32px;
+	width: 24px;
+	height: 24px;
+}
+
+body.admin-bar .jetpack-subscribe-overlay__close {
+	top: 64px;
+}
+
+.jetpack-subscribe-overlay__to-content {
+	display: none;
+	position: fixed;
+	bottom: 64px;
+	left: 0;
+	right: 0;
+	margin: 0 auto;
+
+	a {
+		color: var(--jetpack-subscribe-overlay--text-color);
+	}
```

Comment:
Would be good to avoid `a` styling in CSS and move this to the block markup so that customers can modify it in site editor.

Same for the background colours; styling should get inherited from the theme.

Reply:
fixed!